lib/tasks/panopticon.rake

```diff
@@ -103,7 +104,16 @@ namespace :panopticon do
           registerer.register(artefact)
         rescue GdsApi::HTTPErrorResponse => e
           logger.error "Failed to register /#{edition.slug} with #{e.code}: #{e.error_details}"
+          unregistered_documents << edition.slug
+        rescue
+          logger.error "Failed to register /#{edition.slug}"
+          unregistered_documents << edition.slug
         end
+
+        puts
+        puts "*******************************"
+        puts "Slugs of unregistered documents:"
+        puts unregistered_documents
```

Comment:
Output will probably be more readable if you use `unregistered_documents.join("\n")` here